pkg/session/eventlisteners.go

```diff
@@ -44,7 +44,7 @@ func (sess *Session) listenFunc(ctx context.Context) func(ev interface{}) {
 				req.Initiator = ev.Initiator.Type.String()
 			}
 		case *network.EventLoadingFailed:
-			log.Debugf("Loading failed: %v, %v, Reason; %v, Cancel: %v, %v, %v", ev.RequestID, ev.Type, ev.BlockedReason, ev.Canceled, ev.ErrorText, ev.Timestamp.Time())
+			log.Tracef("Loading failed: %v, %v, Reason; %v, Cancel: %v, %v, %v", ev.RequestID, ev.Type, ev.BlockedReason, ev.Canceled, ev.ErrorText, ev.Timestamp.Time())
```

Comment:
Tror denne bør logges på debug nivå. Gir viktig info for feilsøking.
For å få ned støy, bør vi heller håndtere kjente problemer.